### ISSUE TITLE
Review annotations to GO:0000717 nucleotide-excision repair, DNA duplex unwinding and related terms

### ISSUE THREAD

**Issue by raymond91125** [geneontology/go-annotation#5540]
Dear all, 

The proposal has been made to obsolete BP
GO:0000717 nucleotide-excision repair, DNA duplex unwinding
GO:0044806 G-quadruplex DNA unwinding
GO:1905466 negative regulation of G-quadruplex DNA unwinding
GO:1905467 positive regulation of G-quadruplex DNA unwinding
GO:1905465 regulation of G-quadruplex DNA unwinding
GO:0036292 DNA rewinding
For they are MFs.
see https://github.com/geneontology/go-ontology/issues/29153

Experimental annotations that need to be reviewed are here: 
https://docs.google.com/spreadsheets/d/1erB5WI0Z8FBHCTb3oCat92fDijtsb7u7O9-RV3dPHJE/edit?gid=0#gid=0

Impacted groups: 
ComplexPortal	3
UniProt	12
FlyBase	3 ~done
~TAIR	1~
BHF-UCL	8 -done
PomBase	4
SGD	5

** Group contacts: https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv
----

Mappings that need to be reviewed: (InterPro2GO, UniProt-Keywords, UniRule) 
None

Thanks.

**Comment by ValWood** [comment]
Hi @raymond91125 it seems like we need a term for "guanine quadruplex resolver helicase" (both DNA and RNA)
this particular activity seems to be a major activity of specific helicases including the ReqQ helicases (WRN,BLM ). Pif1 family, Fanconi Anemia Helicase J, DHX36 (RHAU, G4R1) A highly specialized RNA and DNA helicase with robust G4-resolving activity, Dna2, Subunits of the TFIIH transcription/repair complex, these helicases can unwind G4 structures in the context of transcription-coupled repair. and others.

This is useful functional context to retain. I can't find another specific  term to replace  GO:0044806 G-quadruplex DNA unwinding

**Comment by RLovering** [comment]
I agree with Val, I had deleted our  GO:0044806 G-quadruplex DNA unwinding annotations although I had sometimes applied the parent term  GO:0032392 DNA geometric change

**Comment by raymond91125** [comment]
> Hi @raymond91125 it seems like we need a term for "guanine quadruplex resolver helicase" (both DNA and RNA) this particular activity seems to be a major activity of specific helicases including the ReqQ helicases (WRN,BLM ). Pif1 family, Fanconi Anemia Helicase J, DHX36 (RHAU, G4R1) A highly specialized RNA and DNA helicase with robust G4-resolving activity, Dna2, Subunits of the TFIIH transcription/repair complex, these helicases can unwind G4 structures in the context of transcription-coupled repair. and others.
> 
> This is useful functional context to retain. I can't find another specific term to replace GO:0044806 G-quadruplex DNA unwinding

@ValWood Would you make a new term request, please?

**Comment by tberardini** [comment]
TAIR done.

**Comment by hattrill** [comment]
FB done.

**Comment by Antonialock** [comment]
UniProt done

**Comment by sandraorchard** [comment]
Complex Portal done

**Comment by pgaudet** [comment]
Term has been obsoleted. Remaining annotations will appear in error reports.
